### ISSUE TITLE
SpongeForge 1.16.5 Server Crash

### ISSUE THREAD

**Issue by Smile-QWQ** [SpongePowered/Sponge#4007]
### Affected Product(s)

SpongeForge

### Version

1.16.5-36.2.5-8.2.0 RECOMMENDED

### Operating System

Windows 11

### Java Version

1.8.0_352

### Plugins/Mods

```shell
aaa_spongeforge-1.16.5-36.2.5-8.2.0-universal.jar
abnormals_core-1.16.5-3.3.1.jar
abnormals_delight-1.16.5-1.2.1.jar
actuallycompatible-0.1.2.jar
advancedperipherals-1.16.5-0.7.10r.jar
AI-Improvements-1.16.5-0.5.0.jar
aiotbotania-1.16.5-1.8.4.jar
AkashicTome-1.4-16.jar
alexsmobs-1.12.1.jar
Apotheosis-1.16.5-4.8.9A0.jar
appleskin-forge-mc1.16.x-2.5.1.jar
Aquaculture-1.16.5-2.1.23.jar
archers_paradox-1.16.5-1.5.0.2.jar
Architects-Palette-1.16.4-1.1.5.jar
architectury-1.32.68.jar
arsarsenal-1.1.0-1.16.5.jar
ArsEnderStorage-1.16.5-1.1.0.jar
ars_creo-1.16.5-1.1.2.jar
ars_elemental-1.16.5-0.1.6.4.jar
ars_nouveau-1.16.5-1.25.10.jar
Artifacts-1.16.5-2.10.5.jar
astralsorcery-1.16.5-1.14.1.jar
atmospheric-1.16.5-3.1.1.jar
AttributeFix-1.16.5-10.1.4.jar
Atum-1.16.5-2.2.12.jar
AutoRegLib-1.6-49.jar
Babel-1.0.5.jar
badlightningnocookie-1.0.2.jar
BedBenefits-1.16.5-5.0.5.jar
BetterAdvancements-1.16.5-0.1.1.115.jar
betterbiomeblend-1.16.4-1.2.9-forge.jar
BetterBurning-1.16.5-6.0.8.jar
betterendforge-1.16.5-2.5.jar
BetterMineshafts-Forge-1.16.4-2.0.4.jar
BHMenu-Forge-1.16.5-2.4.1.jar
biomeidfixer-1.16.5-1.0.1.s5.jar
BloodMagic-1.16.4-3.1.13-39.jar
Bookshelf-Forge-1.16.5-10.4.33.jar
Botania-1.16.5-420.3.jar
BotaniaAdditionsMC1.16.4-1.0.4.jar
BotanyPots-1.16.5-7.1.41.jar
Bountiful-1.16.4-3.3.1.jar
buildinggadgets-1.16.5-3.8.4-build.25+mc1.16.5.jar
bwncr-1.16.5-3.10.16.jar
byg-1.3.6.jar
caelus-forge-1.16.5-2.1.3.2.jar
cakechomps-forge-1.16.5-5.0.0.3.jar
cant-sleep-clowns-will-eat-me-1.16.4-1.0.0.1.jar
catalogue-1.6.1-1.16.5.jar
catjammies-1.1.0.jar
cc-tweaked-1.16.5-1.101.3.jar
cfm-7.0.0pre22-1.16.3.jar
cherishedworlds-forge-1.16.5-5.1.1.0.jar
chipped-1.16.5-1.2.1-forge.jar
Chisel-MC1.16.5-2.0.1-alpha.4.jar
chiselsandbits-1.0.63.jar
citadel-1.8.1-1.16.5.jar
clickadv-2.3.jar
ClientTweaks_1.16.3-5.3.0.jar
clockout-1.16.4-2.0.0.1.jar
cloth-config-4.17.101-forge.jar
Clumps-6.0.0.28.jar
cobblegenrandomizer-1.16-5.1.2.jar
CodeChickenLib-1.16.5-4.0.7.445-universal.jar
cofh_core-1.16.5-1.5.2.22.jar
collective-1.16.5-5.49.jar
comforts-forge-1.16.5-4.0.1.5.jar
CommonCapabilities-1.16.5-2.8.0.jar
compactcrafting-1.0.0.jar
compactmachines-4.0.0-beta.2.jar
Compote-1.0.2.jar
configswapper-2.7.jar
configured-1.5.4-1.16.5.jar
connectedglass-1.1.9-forge-mc1.16.jar
connectivity-2.4-1.16.5.jar
constructionwand-1.16.5-2.6.jar
Controlling-7.0.0.29.jar
cookiecore-1.16.5-3.3.1-9c789ef.jar
CookingForBlockheads_1.16.5-9.3.4.jar
CosmeticArmorReworked-1.16.5-v5.jar
CraftingTweaks_1.16.5-12.2.1.jar
crashutilities-3.13.jar
create-mc1.16.5_v0.3.2g.jar
createplus-1.16.4_v0.3.2.1.jar
CTM-MC1.16.1-1.1.2.6.jar
Cucumber-1.16.5-4.1.12.jar
culinaryconstruct-forge-1.16.5-4.0.1.0.jar
curioofundying-forge-1.16.5-5.2.0.0.jar
curios-forge-1.16.5-4.1.0.0.jar
Curious Armor Stands-1.16.5-2.2.0.jar
curiouselytra-forge-1.16.5-4.0.2.4.jar
cyclepaintings_1.16.5-2.7.jar
CyclopsCore-1.16.5-1.13.0.jar
dankstorage-1.16.5-3.21.jar
DarkPaintings-1.16.5-6.0.11.jar
DarkUtilities-1.16.5-8.0.14.jar
decorative_blocks-1.16.4-1.7.2.jar
decorative_blocks_abnormals-1.2.jar
DefaultOptions_1.16.5-12.2.1.jar
demagnetize-forge-1.16.2-1.2.2.jar
DiagonalFences-v1.1.1-1.16.5.jar
diet-forge-1.16.5-1.0.4.1.jar
DrawersTooltip-1.16.2-2.1.1.jar
DungeonCrawl-1.16.3-2.2.4.jar
DungeonsArise-1.16.5-2.1.49-beta.jar
dungeons_plus-1.16.5-1.1.5.jar
DustrialDecor-1.3.1.jar
dynviewdist-2.0.jar
eidolon-0.2.7.jar
elevatorid-1.16.5-1.7.13.jar
EmendatusEnigmatica-1.2.16-build.46+mc1.16.5.jar
emojiful-1.16.4-2.1.6.jar
EnchantmentDescriptions-1.16.5-7.1.23.jar
EnderStorage-1.16.5-2.8.0.170-universal.jar
energymeter-1.16.5-1.6.1.jar
engineersdecor-1.16.5-forge-1.1.19.jar
enigmatica-1.16.3-0.1.1.jar
EnigmaticGraves-1.16.5-1.7.7.jar
ensorcellation-1.16.5-1.5.0.4.jar
entangled-1.3.16-forge-mc1.16.jar
EntityCulling-1.16.5-2.1.7.jar
environmental-1.16.5-1.1.1.jar
EquipmentCompare-1.16.5-1.3.2.jar
ERP-1.9.zip
expandability-2.0.1-forge.jar
ExperienceBugFix-1.36.0.2.jar
ExtraStorage-1.16.5-1.6.1.jar
extremeSoundMuffler-3.18_1.16.5.jar
FarmersDelight-1.16.5-0.6.0.jar
FarmingForBlockheads_1.16.5-7.3.1.jar
FastFurnace-1.16.5-4.5.0.jar
FastLeafDecay-v25.2.jar
FastWorkbench-1.16.5-4.6.2.jar
ferritecore-2.1.1-forge.jar
findme-1.16.3-2.2.1.0.jar
FluxNetworks-1.16.5-6.2.1.14.jar
flywheel-1.16-0.2.5.jar
FpsReducer-forge-1.24-mc1.16.5.jar
FramedBlocks-2.16.3.jar
framedcompactdrawers-1.16-2.2.1.jar
FriendlyFire-1.16.5-7.0.6.jar
ftb-backups-2.1.2.2.jar
ftb-chunks-forge-1605.3.4-build.220.jar
ftb-essentials-1605.1.5-build.32.jar
ftb-library-forge-1605.3.4-build.90.jar
ftb-quests-forge-1605.3.7-build.165.jar
ftb-ranks-forge-1605.1.6-build.33.jar
ftb-teams-forge-1605.2.3-build.40.jar
ftb-ultimine-forge-1605.3.1-build.45.jar
fusion-1.1.0-forge-mc1.16.jar
GameMenuModOption-1.16.4-1.8.2.jar
GameStages-Forge-1.16.5-7.3.14.jar
geckolib-forge-1.16.5-3.0.106.jar
glassential-forge-1.16.5-1.1.7.jar
guardvillagers-1.16.5.1.2.6.jar
gunswithoutroses-1.16.5-1.0.9.jar
Iceberg-1.16.5-1.0.45.jar
immersivecooking-1.2.0.jar
ImmersiveEngineering-1.16.5-5.1.0-148.jar
ImmersivePetroleum-1.16.5-3.4.0-20.jar
ImmersivePosts-1.16.5-4.3.0-1.jar
industrial-foregoing-1.16.5-3.2.14.7-16.jar
integratedadditions-1.1.4.jar
IntegratedCrafting-1.16.5-1.0.23.jar
IntegratedDynamics-1.16.5-1.10.15.jar
integratednbt-1.16.4-1.4.2.jar
IntegratedTerminals-1.16.5-1.2.13.jar
IntegratedTunnels-1.16.5-1.8.11.jar
integrated_proxy-1.16-1.0.19.jar
Interactio-1.16.4-3.1.1.jar
interactive_corporea-1.4.jar
InventoryEssentials_1.16.5-2.3.2.jar
InventoryHud_[1.16.2-1.16.5].forge-3.4.1.jar
invtweaks-1.16.4-1.0.1.jar
ironchest-1.16.5-11.2.21.jar
IronJetpacks-1.16.5-4.2.3.jar
item-filters-forge-1605.2.5-build.9.jar
jeed-1.16.5-1.6.jar
jei-1.16.5-7.8.0.1009.jar
JEPB-1.0.0.jar
jmi-1.16.5-0.8.1-47.jar
journeymap-1.16.5-5.8.5p6.jar
justenoughbeacons-3.1-1.16.3.jar
JustEnoughCalculation-1.16.5-3.8.6.jar
JustEnoughProfessions-1.16.5-1.2.2.jar
JustEnoughResources-1.16.5-0.12.2.216.jar
KleeSlabs_1.16.5-9.2.1.jar
kotlinforforge-1.17.0-obf.jar
kubejs-blood-magic-1605.1.1-build.3.jar
kubejs-create-1605.1.4-build.12.jar
kubejs-forge-1605.3.19-build.299.jar
kubejs-immersive-engineering-1605.1.2-build.28.jar
kubejs-mekanism-1605.1.2-build.2.jar
kubejs-thermal-1605.1.4-build.4.jar
LibX-1.16.3-1.0.76.jar
LIST.TXT
littlelogistics-mc1.16.5-v1.0.8.jar
logprot-1.16-1.6.jar
Lollipop-1.16.5-3.2.9.jar
lootr-1.16.5-0.2.18.49.jar
LostTrinkets-1.16.5-0.1.27.jar
magicfeather-1.16.2-3.2.0.jar
Mantle-1.16.5-1.6.157.jar
masonry-1.16.3-0.3.6.jar
MasterfulMachinery-1.16.5-0.1.68-B1105.jar
materialis-1.16.5-2.5.1.jar
MaxHealthFix-Forge-1.16.5-1.0.5.jar
mcjtylib-1.16-5.1.8.jar
mcw-bridges-2.1.0-mc1.16.5forge.jar
mcw-doors-1.1.0forge-mc1.16.5.jar
mcw-fences-1.0.7-mc1.16.5forge.jar
mcw-roofs-2.2.4-mc1.16.5forge.jar
mcw-trapdoors-1.1.2-mc1.16.5forge.jar
mcw-windows-2.2.0-mc1.16.5forge.jar
meetyourfight-1.16.5-1.2.0.jar
Mekanism-1.16.5-10.1.2.457.jar
MekanismGenerators-1.16.5-10.1.2.457.jar
MekanismTools-1.16.5-10.1.2.457.jar
mgui-1.16.5-3.3.0.jar
mightyarchitect-mc1.16.3_v0.5.jar
mininggadgets-1.7.6.jar
modnametooltip_1.16.2-1.15.0.jar
modular-routers-1.16.5-7.5.46.jar
morecfm-1.3.1-1.16.3.jar
moredragoneggs-2.5.jar
moremountedstorages-1.3.0.jar
More_Crafting _Tables_Forge_2.3.0.jar
Morph-o-Tool-1.4-27.jar
MouseTweaks-2.14-mc1.16.2.jar
MythicBotany-1.16.5-1.4.19.jar
NaturesAura-34.3.jar
NaturesCompass-1.16.5-1.9.1-forge.jar
NaturesStarlight-1.2.jar
NBT-Ingredient-Predicate-1.3.jar
NetherPortalFix_1.16.3-7.2.1.jar
nethers_delight-2.1.jar
Notes-1.16.5-1.2.3.jar
observerlib-1.16.5-1.5.3.jar
occultism-1.16.5-1.29.1.jar
occultism_kubejs-1.16.5-0.0.1.jar
overloadedarmorbar-5.1.0.jar
PassthroughSigns-1.16.4-2.3.7.jar
Patchouli-1.16.4-53.3.jar
pedestals-0.8u.jar
personality-1.16.5-1.0.3.jar
Ping-1.16.4-1.6.9.jar
pitg-1.16.5-2.0.1.jar
Placebo-1.16.5-4.7.1.jar
plonk-1.16.5-10.0.4.jar
plugins
pneumaticcraft-repressurized-1.16.5-2.15.12-13.jar
polymorph-forge-1.16.5-0.41.jar
portable_stonecutter-1.16.5-1.3.5.jar
portality-1.16.5-3.2.5.jar
Powah-1.16.5-2.3.16.jar
PrettyPipes-1.9.8.jar
PrettyPipesFluids-1.16.5-0.4.2.jar
projectvibrantjourneys-1.16.5-3.2.10.jar
PuzzlesLib-v1.0.15-1.16.5-Forge.jar
Quark-r2.4-322.jar
quickstack-1.16.5-4.0.3.jar
randompatches-2.4.4-forge.jar
rangedpumps-0.8.2.jar
redstonepen-1.16.5-1.0.11.jar
refinedcooking-1.0.2.jar
refinedcrafterproxy-1.1.0.jar
refinedstorage-1.9.18.jar
refinedstorageaddons-0.7.4.jar
repurposed_structures_forge-3.4.7+1.16.5.jar
ResourcefulBees-1.16.5-0.10.7.jar
restriction-1.16.5-0.1.4.jar
rftoolsbase-1.16-2.1.5.jar
rftoolsbuilder-1.16-3.1.11.jar
rftoolscontrol-1.16-4.0.18.jar
rftoolspower-1.16-3.0.14.jar
rftoolsstorage-1.16-2.0.19.jar
rftoolsutility-1.16-3.1.16.jar
rhino-forge-1605.1.5-build.75.jar
rsgauges-1.16.5-1.2.17.jar
RSInfinityBooster-1.16.5-1.1+13.jar
RSLargePatterns-1.16.5-2.1.0.3.jar
rsrequestify-1.16.5-2.1.6.jar
Runelic-1.16.5-7.0.3.jar
SaveYourPets-1.16.2-1.0.0.7.jar
scaffoldingpower-1.16.5-1.3.0.jar
scannable-1.16.5-forge-1.7.6+00198b7.jar
selene-1.16.5-1.9.0.jar
serverconfigupdater-1.3.2.jar
shiftnscrollonmac-1.16.4-0.0.1.s1.jar
Shrink-1.16.5-1.1.6.jar
simpledelights-1.2.jar
SimpleDiscordRichPresence-1.4.0-build.3+mc1.16.5.jar
simplefarming-1.16.5-1.3.8.jar
simplylight-1.16.5-1.4.2-build.32.jar
smallships-1.16.5-2.0.0-Alpha-0.4.jar
SoL-Carrot-1.16.5-1.10.1.jar
sophisticatedbackpacks-1.16.5-3.15.20.755.jar
sounddeviceoptions-1.4.3.jar
spark-1.9.1-forge.jar
SpawnerFix-1.16.2-1.0.0.2.jar
StorageDrawers-1.16.3-8.5.2.jar
storagedrawersfixes-1.0.0.jar
structure_gel-1.16.5-1.7.8.jar
supermartijn642configlib-1.1.8-forge-mc1.16.jar
supermartijn642corelib-1.1.15-forge-mc1.16.jar
supplementaries-1.16.5-0.18.7.jar
sushigocrafting-1.16.5-0.2.1.jar
swingthroughgrass-1.16.4-1.5.3.jar
tamedsummon-1.0.0.jar
tanknull-2.3-1.16.4.jar
tconplanner-1.16.5-1.2.0.jar
TConstruct-1.16.5-3.3.4.335.jar
TerraForged-1.16.5-0.2.16.jar
tetra-1.16.5-3.20.0.jar
tetranomicon-1.4-1.16.5.jar
theoneprobe-1.16-3.1.7.jar
thermal_expansion-1.16.5-1.5.2.16.jar
thermal_foundation-1.16.5-1.5.2.30.jar
thermal_innovation-1.16.5-1.5.0.4.jar
thermal_locomotion-1.16.5-1.5.0.4.jar
The_Undergarden-1.16.5-0.5.5.jar
time-in-a-bottle-1.1.0.jar
Tips-1.16.5-4.0.18.jar
TipTheScales-1.16.5-3.0.0.15.jar
titanium-1.16.5-3.2.8.8-24.jar
tomeofblood-1.16.5-1.2.3.jar
ToolStats-1.16.5-6.1.6.jar
toomanyglyphs-snapshot-20220223.jar
torchmaster-2.3.8.jar
toughnessbar-6.1.jar
towers_of_the_wild-1.16.3-2.1.0.1.jar
TradingPost-v1.0.2-1.16.5.jar
Transport-1.16.5-3.7.0.jar
trashcans-1.0.18-forge-mc1.16.jar
TrashSlot_1.16.3-12.2.1.jar
TravelAnchors-2.4.jar
trofers-1.16.5-1.1.0.jar
Undergarden-Tetra Patch-1.3.1-1.16.5.jar
upgrade_aquatic-1.16.5-3.1.2.jar
useful_railroads-1.16.5-1.4.6.43.jar
u_team_core-1.16.5-3.2.1.258.jar
valhelsia_core-16.0.15.jar
valhelsia_structures-1.16.5-0.1.6.jar
Waystones_1.16.5-7.6.4.jar
WitherSkeletonTweaks-1.16.5-5.4.1.jar
wits-1.1.1+1.16.5-forge.jar
World Stripper-1.16.5-2.1.0.jar
xnet-1.16-3.0.17.jar
XNetGases-1.16.5-2.3.10.jar
YungsApi-1.16.4-Forge-13.jar
```

### Describe the bug

Enigmatica 6 Expert-1.9.0
abnormals_core has been replaced with unoffical patched version, but still can't start the server

The following mods have been removed：
observable
lazydfu
roadrunner

### Link to logs

crash-report & log：https://paste.gg/p/anonymous/7b65b66473cc4571b8b9ad46692bf02f

**Comment by gabizou** [comment]
Try this jar:
[SpongeForge Production Jar.zip](https://github.com/SpongePowered/Sponge/files/15288422/SpongeForge.Production.Jar.zip)

**Comment by Smile-QWQ** [comment]
> Try this jar: [SpongeForge Production Jar.zip](https://github.com/SpongePowered/Sponge/files/15288422/SpongeForge.Production.Jar.zip)

Thanks, I tried this jar and the server starts up fine

**Comment by gabizou** [comment]
Fixed with spongeforge-1.16.5-36.2.5-8.2.1-RC1587
